packages/expo-splash-screen-command/README.md

```diff
@@ -0,0 +1,60 @@
+# expo-splash-screen-command
+
+This module provides supporting installation command for [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen) module.
+You can use this command to configure your native iOS and Android project according to your needs without opening XCode or Android Studio.
+
+## Content
+
+- [📜	CHANGELOG](./CHANGELOG.md)
+- [🚀 Features](#-features)
+- [🗒 Usage](#-usage)
+- [🖥 Installation](#-installation)
+- [👏 Contributing](#-contributing)
+
+## 🚀 Features
+
+### 📱 iOS
+
+- configures background color for native splash screen
+- configures [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen) to show given `.png` image
+- supports [`CONTAIN`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#contain-resize-mode) and [`COVER`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#cover-resize-mode) modes from [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen)
+
+### 🤖 Android
+
+- configures background color for native splash screen
+- configures `expo-splash-screen` to show given `.png` image
+- supports [`CONTAIN`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#contain-resize-mode), [`COVER`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#cover-resize-mode) and [`NATIVE`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen#native-resize-mode) modes from [`expo-splash-screen`](https://github.com/expo/expo/tree/master/packages/expo-splash-screen)
+
+## 🗒 Usage
+
+Command syntax:
+```
+yarn run expo-splash-screen --mode "contain" (default) | "cover" | "native" (only on Android) --platform "all" (default) | "ios" | "android" <backgroundColor> (required) [imagePath] (optional)
+```
+
+To see all the available options:
+```
+yarn run expo-splash-screen --help
+```
+### Alternatives to `yarn`
+
+Instead of 
+```
+yarn run expo-splash-screen
+```
+you can go with:
+```
+npm run expo-splash-screen
+```
+or
+```
+npx expo-splash-screen
```

Comment:
I think we don't want to have to much stuff about yarn alternatives, we decided to prefer yarn everywhere and I believe most other libs just use one of these in readmes